snippets/base/storage.py

```diff
@@ -10,8 +10,17 @@
 
 
 class OverwriteStorage(FileSystemStorage):
+    """
+    Comes from http://www.djangosnippets.org/snippets/976/
+    (even if it already exists in S3Storage for ages)
```

Comment:
I'm confused about the mention of S3Storage here. This `OverwriteStorage` class doesn't seem to have anything to do with the S3 storage stuff.

Reply:
Fair comment. Not sure why this is there. I just refactored / copy-pasted from `snippets.storage` to `snippets.base.storage`. Will remove this comment make things clear.